src/metatrain/experimental/pet/trainer.py

```diff
@@ -687,6 +690,12 @@ def save_model(model_name, model_keeper):
             )
             summary += f"at epoch {forces_rmse_model_keeper.best_epoch}\n"
 
+            if forces_mae_model_keeper.best_error < self.best_loss:
+                self.best_loss = forces_mae_model_keeper.best_error
```

Comment:
Here we're comparing `self.best_loss` (which is not a loss but potentially an energy(?)) with the force error